packages/s/sol2/xmake.lua

```diff
@@ -29,7 +29,9 @@ package("sol2")
     end)
 
     on_install(function (package)
-        import("package.tools.cmake").install(package)
+		if not package:is_plat("iphoneos") then
+        	import("package.tools.cmake").install(package)
+		end
```

Comment:
incorrect fix, you should not do it.

Reply:
changed in commit `exclude iphoneos attempt 2`